File: Java/백준/Bronze/13484. Tarifa/Tarifa.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int lim = in.nextInt();
        int n = in.nextInt();
        int cnt = 0;
        for (int i=0;i<n;i++){
            cnt += in.nextInt();
        }
        System.out.println((n+1)*lim-cnt);
    }
}

File: C#/백준/Bronze/13484. Tarifa/Tarifa.cs
using System;

namespace q{
    class w{
        static void Main(){
            int lim = int.Parse(Console.ReadLine());
            int n = int.Parse(Console.ReadLine());
            int cnt = 0;
            for (int i=0;i<n;i++){
                cnt += int.Parse(Console.ReadLine());
            }
            Console.WriteLine((n+1)*lim-cnt);
        }
    }
}